src/routers/users.routes.js

```diff
@@ -46,6 +47,29 @@ const router = express.Router();
  */
 
 router.post('/login', userController.login);
+/**
+ * @swagger
+ * /api/v1/auth/logout:
+ *   get:
+ *     tags:
+ *       - Log-out
+ *     name: logout
+ *     summary: Log out auth-user
+ *     produces:
+ *       - application/json
+ *     consumes:
+ *       - application/json
+ *     parameters:
+ *       - name: x-access-token
+ *         in: header
+ *         description: jwt token of the user
+ *     responses:
+ *       '200':
+ *             description: User is successfully logged out
+ * */
+
+router.get('/logout', checkUser, userController.logout);  
+
```

Comment:
why are these routes in two places

also in auth.routes.js

Reply:
done